src/common/Exceptions.ts

```diff
@@ -1,8 +1,13 @@
 import { AxiosError, AxiosResponse } from 'axios'
 
-export class ArgumentNullException extends Error {
+export class DevopnessSdkError extends Error {
```

Comment:
Maybe this class should be just called `SdkError`? It's all in the Devopness module anyways.

Reply:
Good call! Done!